concrete/blocks/express_form/controller.php

```diff
@@ -514,7 +510,7 @@ public function save($data)
                                 $key->setAttributeType($type);
                                 $settings = $control->getAttributeKey()->getAttributeKeySettings();
                                 $settings->setAttributeKey($key);
-                                $settings = $settings->mergeAndPersist($entityManager);
+                                $settings->mergeAndPersist($entityManager);
```

Comment:
You sure about this change? mergeAndPersist specifically returns a value that is now merged in the entity manager:

    $detachedEntity = unserialize($serializedEntity); // some detached entity
    $entity = $em->merge($detachedEntity);

Please change this pull request so that it only makes the obvious changes like removing needless facades, actual code styling cleanup, and removing unused imports, etc... Anything else and you might be misunderstanding what the original code does.

Reply:
I removed it because `$settings` isn't used later on in the code (afaik). I'll add it back if that's what you prefer.